jerry-core/ecma/builtin-objects/ecma-builtin-array.inc.h

```diff
@@ -19,6 +19,8 @@
 
 #include "ecma-builtin-helpers-macro-defines.inc.h"
 
+#ifndef CONFIG_DISABLE_ARRAY_BUILTIN
+
```

Comment:
If I understand correctly this files are only included if CONFIG_DISABLE_ARRAY_BUILTIN is not present. So adding it here seems pointless. Perhaps you can add an # error if present.